packages/block-editor/src/components/iframe/content.scss

```diff
@@ -5,18 +5,20 @@
 
 .block-editor-iframe__html {
 	transform-origin: top center;
-	@include editor-canvas-resize-animation;
+	transition: all 0.4s cubic-bezier(0.46, 0.03, 0.52, 0.96), transform 0s;
```

Comment:
We could use a comment about the numbers even like, "this is what works" 😄

Reply:
It was just removing the mixin since it's only used once: https://github.com/WordPress/gutenberg/pull/66034/files#diff-3014a931b478b662a3c554c1306e3b548da2c2edd5b32114395b329404f8f50aL40